packages/drpc/src/DrpcModule.ts

```diff
@@ -0,0 +1,31 @@
+import type { FeatureRegistry, DependencyManager, Module } from '@credo-ts/core'
+
+import { Protocol } from '@credo-ts/core'
+
+import { DrpcApi } from './DrpcApi'
+import { DrpcRole } from './DrpcRole'
+import { DrpcMessageRepository } from './repository'
+import { DrpcService } from './services'
+
+export class DrpcModule implements Module {
+  public readonly api = DrpcApi
+
+  /**
+   * Registers the dependencies of the drpc message module on the dependency manager.
+   */
+  public register(dependencyManager: DependencyManager, featureRegistry: FeatureRegistry) {
```

Comment:
WOuld be good to add the `experimental` log in the beginning. And we can remove it after some public use (like at 0.6.0 or something).

Reply:
I think it is important to add this mark, especially because the DRPC protocol itself is not yet merged into Aries RFCs repo, so it's likely to have some updates at some point.